pcsx2/CDVD/InputIsoFile.cpp

```diff
@@ -160,6 +160,10 @@ int InputIsoFile::FinishRead3(u8* dst, uint mode)
 InputIsoFile::InputIsoFile()
 {
 	_init();
+	m_reader = NULL;
```

Comment:
m_reader could be done in the init phase? In this case we could remove the one in the destructor.

To be investigated for m_current_count and ReadUnit.